src/gui/AbInitio3DPanel.cpp

```diff
@@ -2644,10 +2644,14 @@ void AbInitioManager::ProcessAllJobsFinished()
 		input_refinement->resolution_statistics_pixel_size = active_pixel_size;
 		input_refinement->resolution_statistics_box_size = check_file.ReturnXSize();
 		input_refinement->SetAllPixelSizes(active_pixel_size);
-		input_refinement->SetAllVoltages(300);
-		input_refinement->SetAllCs(2.7);
-		input_refinement->SetAllAmplitudeContrast(0.07);
-		input_refinement->SetAssignedSubsetToEvenOdd();
+
+		if (active_use_classums == true) // in this case input refinement and output refinement should be the same
+		{
+			input_refinement->SetAllVoltages(300);
+			input_refinement->SetAllCs(2.7);
+			input_refinement->SetAllAmplitudeContrast(0.07);
+			input_refinement->SetAssignedSubsetToEvenOdd();
+		}
```

Comment:
I do not recall why we have to set these attributes like this. Shouldn't they be copied from the `active_refinement_package` instead?

Reply:
This is for the case of ab-inito from class averages.  In this case, the defocus values is set to 0.0 which should make CTF::Evaluate return a constant for the CTF.  I included setting voltage to 300 and Cs to 2.7 just in case having weird values there caused problems.  I could use the voltage and cs from the refinement package, but it shouldn't make a difference in this case.

Reply:
Ah yes, OK. Thanks. This is after the stack was created, so we're talking about the summed images, not the raw images.

Reply:
Yup, they are already CTF corrected.